src/features/tokens/metadata.ts

```diff
@@ -1,7 +1,9 @@
 import { EvmTokenAdapter, ITokenAdapter, TokenType } from '@hyperlane-xyz/sdk';
 import { ProtocolType } from '@hyperlane-xyz/utils';
 
-import { tokenList } from '../../consts/tokens';
+import { tokenList as TokensTS } from '../../consts/tokens';
+import TokensJson from '../../consts/tokens.json';
+import TokensYaml from '../../consts/tokens.yaml';
```

Comment:
For warp token artifacts, they are not currently being output with tokens.yaml as the file name right?

Reply:
The CLI artifact file name is `warp-ui-token-config.json` but I don't recommend we use that long name here